backend/engine/plugins/gitlab_repo_health/README.md

```diff
@@ -0,0 +1,310 @@
+# gitlab_repo_health
+
+Artemis Plugin for checking that GitLab repositories are configured to meet a security baseline
+
+## Testing
+
+Tests are included in `artemis/engine/tests`
+
+## Configuration
+
+Configurations are of the following format:
+
+```json
+{
+    "name": "example",
+    "description": "An example config. It is not strictly valid as it has comments and placeholders",
+    "version": "1.0.0",
+    "rules": [
+        {
+            # Type is required. Every object in the config must have a type
```

Comment:
To embed comments in example JSON, we can use the `jsonc` language type (i.e. instead of `json`).

This allows inline JS-style comments, e.g.:

```jsonc
{
    "foo": "bar",
    // This value is very important!
    "baz": 123
}
```